spec/zip_writer_spec.rb

```diff
@@ -1,44 +1,38 @@
 require_relative './spec_helper'
-require 'rspec/mocks'
 require 'zip_writer'
 require 'zip'
-require 'pry'
+require 'pathname'
```

Comment:
Most of the changes in this file are to support passing Pathnames to ZipWriter.  Other changes are commented.